src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -403,6 +409,45 @@ public object VisitTable(TableExpression tableExpression, SearchOptions context)
                         StringBuilder.Append("FROM ").AppendLine(includeCte);
                     }
 
+                    break;
+                case TableExpressionKind.Sort:
+
+                    var (searchParamaterInfo, s) = context.GetFirstSortSupportedParam();
+
+                    // the normalized generator holds the data for the search table. TODO: use it.
+                    if (searchParamaterInfo.Type == ValueSets.SearchParamType.Date)
+                    {
+                        if (tableExpression.ChainLevel == 0)
+                        {
+                            StringBuilder.Append("SELECT ").Append(VLatest.Resource.ResourceSurrogateId, null).Append(" AS Sid1, ")
+                                .Append(VLatest.DateTimeSearchParam.StartDateTime, null).AppendLine(" as SortExpr")
```

Comment:
as SortExpr -> AS Sdt?

Reply:
@rbans96 what does `Sdt` stands for?
I chose `SortExpr` so it can fit any sort type in the future (i.e. number, text etc)